front/app/templates/builds.hbs

```diff
@@ -20,12 +18,8 @@
         {{#each model.recent_failures as |build|}}
 
         <tr>
-            <td>
-                <a href="http://buildbot.rust-lang.org/builders/{{build.builder_name}}/builds/{{build.number}}" target="_blank">
-                    {{build.number}}
-                </a>
-            </td>
-            <td>{{build.builder_name}}</td>
+            <td>{{build.number}}</td>
```

Comment:
Not especially -- we can always add a `build_url` column to database and use it here. Otherwise, IIRC, we can get the Travis URL from some string interpolation, but it's trickier with Appveyor (Appveyor has a weird distinction between build numbers and build versions).